docs/ref-ergo-spec.md

```diff
@@ -218,16 +218,16 @@ Precedence determines the order of operations in expressions with operators of d
 Table of operators in Ergo with their associativity and precedence from highest to lowest:
 
 **Order** | **Operator(s)** | **Description** | **Associativity**
---- | --- | --- | ---
-1 | . <br> ?. | field access <br> field access of optional type | left to right
-2 | [] | array index access | right to left
-3 | ! | logical not | right to left
-4 | \- | arithmetic negation | right to left
-5 | ++ | string concatenation | left to right
-6 | ^ | floating point number power | left to right
-7 | \* <br> / <br> % | multiplication <br> division <br> remainder | left to right
-8 | \+ <br> - | addition <br> subtraction | left to right
-9 | ?? | default value of optional type | left to right
-10 | and | logical conjunction | left to right
-11 | or | logical disjunction | left to right
-12 | < <br> > <br> <= <br> >= <br> = <br> != | less than <br> greater than <br> less or equal <br> greater or equal <br> equal <br> not equal | left to right
+--- | --- | --- | --- 1 | . <br /> ?. | field access <br /> field access of
```

Comment:
The table structure has been broken here an no longer renders properly. Do you understand why?

Reply:
Yes, I used the automatic migration cli command provided by docusaurus ```npx @docusaurus/migrate migrate --mdx --page ./v1-website ./v2-website``` . Since, this is still in alpha, so it replaced the table structure incorrectly, leading to it being broken. I'll review any broken tables and similar files and fix them in the upcoming commits.

Reply:
I fixed all docs files after reviewing and comparing them in a6f0627926cce81b5b0389d9eb17b1f1ee7b4437. The error was caused because in JSX, <br> tag now needs to be closed.

Reply:
Hi @prakharsr, no problem and thanks for the update. I'll review again this weekend.